what.html

```diff
@@ -7,41 +7,110 @@
+                             <img src="https://pingpong90.github.io/images/nza_logo.jpg"alt="NZA Logo">
+                        </a>
+                    </div>
+                    
+                    <!-- Navigation List -->
+                    <hr class=hr1>
+                                    
+                    <button><a href="/who.html">   WHO WE ARE |</a></button>
+                    <button><a href="/what.html"> WHAT WE DO |</a></button>
+                    <button><a href="https://nzalaw.com/news/"> NEWS & EVENTS |</a></button>
+                    <button><a href="https://nzalaw.com/practice-group/commercial/"> WHERE WE WORK |</a></button>
+                    <button><a href="https://nzalaw.com/contact/">CONTACT US</a></button>
+                        
+                    <hr class=hr2>
+                </Header>
     
+                <!-- What we are Text  Monica-->
+                <div class="who-text">
+                    <p>What we do at Nielsen Zehe & Antas PC is prevent our client's problems in some cases, solve their problems in other cases, and ultimately, if the problems cannot be prevented or solved, prepare to take the problems to trial and WIN. Our attorneys, who view themselves as problem solvers, have tried more cases to a jury or judge than most law firms collectively. Whether the problem is a lawsuit, busienss deal, or formation of a new business, we have the skill, experience, and business acumen to help our clients.</p>
+                </div>
+        
+                <!-- What We Are Box Stephanie-->
+                <div class="who-box">
+                    <h3> WHAT WE DO</h3>
+                    <hr>
+                    <h4>Subrogation</h4>
+                    <ul class="sub">
+                        <li> Property</li><br>
+                        <li>Workers Compensation</li>
+                    </ul>
+                    <h4>Litigation</h4>
+                    <ul class="lit">    
+                        <li>Appellate<li><br>
+                        <li>Commercial</li><br>
+                        <li>Defense</li><br>
+                        <li>Labor/Employment</li>
+                    </ul>
+                    <h4>Business</h4>
+                    <ul class="biz">
+                        <li>Brockerage Agreement</li><br>
+                        <li>Business Formation</li><br>
+                        <li>Business Related Contracts</li><br>
+                        <li>Labor/Employment</li>
+                    </ul>
+                    <h4>Transportation</h4>
+                    <p class="aligncenter">
+                    <img src="https://images.pexels.com/photos/225502/pexels-photo-225502.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940" id="redbox_pic" width="150" height="84"></p>
+                    <b><h4>Attorneys</h4></b>
+                    <h5>Meet our prestigious partners and esteemed associates.</h5>  
+                </div>
```

Comment:
This looks great too. Are these the changes you were working on with @secrissey?